cmd/gpu_nfdhook/labeler.go

```diff
@@ -114,17 +114,18 @@ func fallback() uint64 {
 }
 
 // getMemoryAmount reads the GPU memory amount from the system.
-func (l *labeler) getMemoryAmount(gpuName string) uint64 {
+func (l *labeler) getMemoryAmount(gpuName string) (uint64, uint64) {
```

Comment:
Now the function name doesn't fully reflect what it actually returns. I guess the return values should be documented somehow for better readability. Either in the doc string or by giving them names. 

The rest looks good!